<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebCameraSettingPage</class>
 <widget class="QWidget" name="WebCameraSettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebCameraSettingPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>519</width>
       <height>380</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_titleBar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_webCameraSettingTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>mulitCameraSetting</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_minSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>minSize</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_close">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>close</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>56</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>24</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>13</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_pos1">
                <property name="maximumSize">
                 <size>
                  <width>72</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="text">
                 <string>pos1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_pos2">
                <property name="maximumSize">
                 <size>
                  <width>72</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="text">
                 <string>pos2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_pos3">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>72</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="text">
                 <string>pos3</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="UserSeatVideoWnd" name="videoWidget" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>242</width>
                <height>182</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>242</width>
                <height>182</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>18</number>
              </property>
              <property name="bottomMargin">
               <number>15</number>
              </property>
              <item>
               <widget class="QLabel" name="label_videoFormat">
                <property name="text">
                 <string>videoFormat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_4_3">
                <property name="text">
                 <string>4:3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_16_9">
                <property name="text">
                 <string>16:9</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_webCameraList">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_code">
            <property name="text">
             <string>code:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <widget class="QLineEdit" name="editCameraCode"/>
            </item>
            <item>
             <widget class="ClickableLabel" name="label_codeIcon">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_classroomSettingHint">
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_QRCodeDetail">
            <property name="text">
             <string>QRCodeDetail</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UserSeatVideoWnd</class>
   <extends>QWidget</extends>
   <header location="global">userseatvideownd.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>WebCameraSettingPage</receiver>
   <slot>closeBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pos1</sender>
   <signal>clicked()</signal>
   <receiver>WebCameraSettingPage</receiver>
   <slot>posOneBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pos2</sender>
   <signal>clicked()</signal>
   <receiver>WebCameraSettingPage</receiver>
   <slot>posTwoBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pos3</sender>
   <signal>clicked()</signal>
   <receiver>WebCameraSettingPage</receiver>
   <slot>posThrBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_4_3</sender>
   <signal>clicked()</signal>
   <receiver>WebCameraSettingPage</receiver>
   <slot>vdsOneBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_16_9</sender>
   <signal>clicked()</signal>
   <receiver>WebCameraSettingPage</receiver>
   <slot>vdsTwoBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeBtnClicked()</slot>
  <slot>posOneBtnClicked()</slot>
  <slot>posTwoBtnClicked()</slot>
  <slot>posThrBtnClicked()</slot>
  <slot>vdsOneBtnClicked()</slot>
  <slot>vdsTwoBtnClicked()</slot>
 </slots>
</ui>
